### PR TITLE
use gulp-load-plugins

### PR DESCRIPTION
**Pull request by artitudinale1**
So left out plugins : `browserify` , `vinyl`, `browser-sync` and `wiredep`

### CODE REVIEW

File: gulp-tasks/deploy.js [hunk 1]
```diff
@@ -1,23 +1,16 @@
...
-    .pipe(gulpif('*.css', rev()))
-    .pipe(revReplace())
-    .pipe(deploy());
+    .pipe(gulpif('*.js', $.replace('var localData = true;', 'var localData = false;')))
+    .pipe(gulpif('*.js', $.uglify({ mangle: false }))) // Minify JavaScript
+    .pipe(gulpif('*.js', $.rev()))
+    .pipe(gulpif('*.css', $.cleanCss())) // Minify CSS
+    .pipe(gulpif('*.css', $.rev()))
+    .pipe($.revReplace())
+    .pipe($.deploy());
```
Comment (chris-alexander) [inline gulp-tasks/deploy.js]: `gulpif` should be `$.if`

**Comment by artitudinale1** [comment]
Updated  -is this ready to be merged in?

File: gulp-tasks/inject.js [hunk 2]
```diff
@@ -1,6 +1,5 @@
 var gulp = require('gulp');
-var inject = require('gulp-inject'); // Injects our front-end JS and CSS files
-var wiredep = require('wiredep').stream; // Injects bower front-end dependencies
```
Comment (chris-alexander) [inline gulp-tasks/inject.js:3]: we still need `wiredep` here. it isn't a gulp plugin :)

File: gulp-tasks/script.js [hunk 3]
```diff
@@ -1,12 +1,11 @@
 var gulp = require('gulp');
-var jshint = require('gulp-jshint');
-var stylish = require('jshint-stylish');
```
Comment (chris-alexander) [inline gulp-tasks/script.js:3]: `jshint-stylish` isn't a gulp plugin

File: gulp-tasks/deploy.js [hunk 4]
```diff
@@ -1,23 +1,16 @@
...
-    .pipe(gulpif('*.css', rev()))
-    .pipe(revReplace())
-    .pipe(deploy());
+    .pipe($.if('*.js', $.replace('var localData = true;', 'var localData = false;')))
+    .pipe($.if('*.js', $.uglify({ mangle: false }))) // Minify JavaScript
+    .pipe($.if('*.js', $.rev()))
+    .pipe($.if('*.css', $.cleanCss())) // Minify CSS
+    .pipe($.if('*.css', $.rev()))
+    .pipe($.revReplace())
+    .pipe($.deploy());
```
Comment (chris-alexander) [inline gulp-tasks/deploy.js]: sorry found another one! Please change `deploy` to `$.ghPages`

**Comment by chris-alexander** [comment]
looks good 👍

**Pull request merged**